### ISSUE TITLE
Improve the ToC interface

### ISSUE THREAD

**Issue by joncamfield** [OpenInternet/SAFETAG-website#3]
The toc.js pulls from the header structure, which makes it easy to auto-generate, but has a few usability challenges:

* It uses numerical references, which change if the order of the guide changes
* References to specific anchors from external links don't work, as the toc is generated on page load
* The pop-down action is really annoying and hard to navigate.  A click-to-open (instead of hover) or delay after the hover action would maybe help?

**Comment by blazman** [comment]
Yes, I also find the pop-down to be annoying. Are those options possible via toc.js? And also thinking about permalinks so that we can hyperlink related content instead of just pulling in snippets of it. I tried to link directly to an activity during a training and was wondering why it didn't work before I saw this.
